inventory/idm/hosts

```diff
@@ -0,0 +1,8 @@
+[idm]
+idm.example.com
```

Comment:
need to remove this line as it's relying on dynamic inventory rather than a static set of hosts